packages/plugins/GoPlusSecurity/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { registerPlugin } from '@masknet/plugin-infra'
 import { base } from './base'
+export * from './messages'
```

Comment:
DO NOT export from the `index.ts`. `index.ts` of a plugin should only provide the register of a plugin. Please import like `@pakcage/name/messages` and define it in the `package.json` using `exports` field and `typeVersions`.